app/javascript/dashboard/composables/useFontSize.js

```diff
@@ -0,0 +1,157 @@
+/**
+ * @file useFontSize.js
+ * @description A composable for managing font size settings throughout the application.
+ * This handles font size selection, application to the DOM, and persistence in user settings.
+ */
+
+import { computed, ref, watch } from 'vue';
+import { useUISettings } from 'dashboard/composables/useUISettings';
+import { useAlert } from 'dashboard/composables';
+import { useI18n } from 'vue-i18n';
+
+/**
+ * Mapping of semantic font size names to their pixel values
+ * @type {Object}
+ */
+const FONT_SIZE_MAP = {
+  smaller: '14px',
+  small: '15px',
+  default: null, // null means use browser default
```

Comment:
```suggestion
  default: '16px'
```

Let's explicitly set the default value. Either way tailwind includes browser resets that will set this for us, so we won't be defaulting to browser default